Source/WebCore/Modules/reporting/ReportingClient.h

```diff
@@ -41,7 +41,7 @@ struct WEBCORE_EXPORT ReportingClient {
 
     virtual void notifyReportObservers(Ref<Report>&&) = 0;
     virtual String endpointURIForToken(const String&) const = 0;
-    virtual void sendReportToEndpoints(const URL& baseURL, const Vector<String>& endpointURIs, const Vector<String>& endpointTokens, Ref<FormData>&& report, ViolationReportType) = 0;
+    virtual void sendReportToEndpoints(const URL& baseURL, const std::span<const String> endpointURIs, const std::span<const String> endpointTokens, Ref<FormData>&& report, ViolationReportType) = 0;
```

Comment:
we should remove the `const` outside the span. There is no good reason for the parameter to be const here.